src/guides/v2.3/graphql/queries/directory-countries.md

```diff
@@ -34,7 +34,7 @@ query {
 
 **Response:**
 
-In this example, the response is intentionally truncated.
+In this example, the response is intentionally truncated. Here `available_regions` attribute value will be null if country doesn't have any regions available otherwise it will be an array of regions for the respective country. 
```

Comment:
```suggestion
In this example, the response is intentionally truncated. The `available_regions` attribute value will be null if the country does not have any regions available, otherwise it will be an array of regions for the respective country.
```